src/java.base/share/classes/java/io/ProxyingConsole.java

```diff
@@ -83,9 +85,17 @@ public Reader reader() {
      * {@inheritDoc}
      */
     @Override
-    public Console format(String fmt, Object ... args) {
+    public Console format(String format, Object ... args) {
+        return format(Locale.getDefault(Locale.Category.FORMAT), format, args);
```

Comment:
Given the number of calls to Locale.getDefault(Locale.Category.FORMAT) it might be worthwhile to cache that in the Proxying Console or in the JdkConsoleImpl.

Reply:
Initially, I thought about it but did not cache it here because it is cached in the `Locale` class, and the call returns the cached value.